chaos_genius/views/alert_view.py

```diff
@@ -1,96 +1,125 @@
 # -*- coding: utf-8 -*-
 """alert data view."""
-from chaos_genius.databases.models.alert_model import Alert
-from datetime import datetime, timedelta
-
-from flask import (
-    Blueprint,
-    current_app,
-    flash,
-    redirect,
-    render_template,
-    request,
-    url_for,
-    jsonify
-)
-from chaos_genius.controllers.alert_controller import get_alert_list, get_alert_info
+
+import logging
+
+from flask.blueprints import Blueprint
+from flask.globals import request
+from flask.json import jsonify
+
+from chaos_genius.controllers.alert_controller import get_alert_info, get_alert_list
 from chaos_genius.databases.db_utils import chech_editable_field
+from chaos_genius.databases.models.alert_model import Alert
 
 blueprint = Blueprint("alert", __name__)
+logger = logging.getLogger(__name__)
 
 
-@blueprint.route("/", methods=["GET"]) # TODO: Remove this
+@blueprint.route("/", methods=["GET"])  # TODO: Remove this
 @blueprint.route("", methods=["GET"])
 def list_alert():
     """List the alert data."""
     results = get_alert_list()
     return jsonify({"data": results})
 
+
 @blueprint.route("/<int:alert_id>/get-info", methods=["GET"])
 def get_alert_info_by_id(alert_id):
-    """get alert details."""
+    """Info of a single alert by ID."""
     status, message = "", ""
     data = None
     try:
         alert_obj = get_alert_info(alert_id)
         data = alert_obj
-        status = "success" 
+        status = "success"
         message = "Alert found Successfully"
-    except Exception as err:
+    except Exception as err:  # noqa: B902
```

Comment:
Is there are particular error type we are expecting here?

Reply:
A lot of the exception blocks have no specified error.

Reply:
These are meant to be a catch-all. Even if there were specific exceptions to be caught, we would need a catch-all at the end anyway.

We aren't handling any exceptions in a special way because there isn't anything special to be done in any case (for example, on some exceptions we would need to do a rollback or add a failure checkpoint somewhere. There is no need to do any such thing in these APIs).

Though, it can be argued that it would be a better practice to not have a catch-all and fail with a 500 server error if there was an unhandled exception. In that case, all failure modes need to known and handled specially - input validation, DB validation, etc. - with the correct error message instead of an exception message. When everything else is handled, an exception is a true "server error". But because we have chosen to use a catch-all in all of our APIs instead, I did not change this behavior and made flake8 ignore the error.